packages/yourloops/test/components/headers/main-header.test.tsx

```diff
@@ -0,0 +1,137 @@
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React from "react";
+import { render, unmountComponentAtNode } from "react-dom";
+import { act } from "react-dom/test-utils";
+import { Router } from "react-router-dom";
+import { createMemoryHistory } from "history";
+import { v4 as uuidv4 } from "uuid";
+
+import { AuthContextProvider, Session } from "../../../lib/auth";
+import { INotification, NotificationContextProvider, NotificationType } from "../../../lib/notifications";
+import { createAuthHookStubs } from "../../lib/auth/utils";
+import { notificationAPIStub } from "../../lib/notifications/utils";
+import { TeamContextProvider } from "../../../lib/team";
+import { teamAPI } from "../../lib/team/utils";
+import { loggedInUsers } from "../../common";
+import { triggerMouseClick } from "../../common/utils";
+import MainHeader from "../../../components/header-bars/main-header";
+
+describe("Main Header", () => {
+  let container: HTMLElement | null = null;
+  const history = createMemoryHistory({ initialEntries: ["/preferences"] });
+  const { hcpSession } = loggedInUsers;
+
+  async function mountComponent(session: Session): Promise<void> {
+    const authContext = createAuthHookStubs(session);
+
+    await act(() => {
+      return new Promise((resolve) => {
+        render(
+          <Router history={history}>
+            <AuthContextProvider value={authContext}>
+              <TeamContextProvider teamAPI={teamAPI}>
+                <NotificationContextProvider api={notificationAPIStub}>
+                  <MainHeader />
+                </NotificationContextProvider>
+              </TeamContextProvider>
+            </AuthContextProvider>
+          </Router>, container, resolve);
+      });
+    });
+  }
+
+  beforeEach(() => {
+    container = document.createElement("div");
+    document.body.appendChild(container);
+  });
+
+  afterEach(() => {
+    if (container) {
+      unmountComponentAtNode(container);
+      container.remove();
+      container = null;
+    }
+  });
+
+  it("Should be able to render", async () => {
```

Comment:
opt: maybe this test is not useful as other tests seem to be based on the fact that the component renders :slightly_smiling_face: